inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndex.java

```diff
@@ -687,11 +702,16 @@ private void indexAllDocuments()
         for (SourceDocument document : documentService.listSourceDocuments(project)) {
             try {
                 indexDocument(document, documentService.createOrReadInitialCas(document));
+                sd++;
             }
             catch (IOException e) {
                 log.error("Unable to index source document", e);
             }
         }
+        
```

Comment:
I would catch the `IOException` in `indexDocument` and log it there once instead of catching it it a lot of different places.

Reply:
It's important to know and log the different points in which it may occur.

Reply:
You know the points because of the stack trace in the exception you log.

Reply:
OK